api/src/services/commentService.ts

```diff
@@ -6,18 +6,50 @@ import { CommentParentNotFoundError } from "../errors/CommentParentNotFoundError
 import { CommentNotFoundError } from "../errors/CommentNotFoundError";
 import { CommentNotOwnedError } from "../errors/CommentNotOwnedError";
 import { ValidationUtils } from "../../src/utils/validationUtils";
-import { CommentParser } from "../models/comment";
+import { Comment, CommentParser } from "../models/comment";
+import NotificationCreationService from "./notificationCreationService";
+import {
+    NotificationEntity,
+    NotificationType,
+} from "../entity/notification.entity";
+
+class ReplyNotification extends NotificationCreationService {
+    private comment: Comment;
+    private parentUserId: number;
+
+    public constructor(
+        notificationRepository: Repository<NotificationEntity>,
+        comment: Comment,
+        parentUserId: number
+    ) {
+        super(notificationRepository);
+
+        this.comment = comment;
+        this.parentUserId = parentUserId;
+    }
+
+    getNotificationText(): string {
+        return `User ${this.parentUserId} replied to your comment made at ${this.comment.mediaId}.`;
+    }
+
+    getNotificationType(): NotificationType {
+        return NotificationType.COMMENT_REPLY;
+    }
+}
```

Comment:
WDYT about having each class in a separate file?

Reply:
Então, eu coloquei no mesmo arquivo por dois motivos principais
- Esse tipo de notificação não será usado em nenhum outro lugar
- Se fossemos criar um arquivo para colocar essa classe, em que pasta ele ficaria? Não me parece um 'service' ou um 'utils'

Se vc achar que podemos criar mais uma subpasta dentro do api/src para colocar as classes que herdam de NotificationCrationService, posso fazer isso

Reply:
Muitas das outras classes não são usadas em nenhum outro lugar. Acho que é legal manter cada uma em um arquivo pra organização mesmo.

Mas como é uma classe filha da NotificationCreationService, acho que se enquadra como service sim. Por mim poderia deixar direto nessa pasta mesmo

Reply:
ok